src/Rollbar.php

```diff
@@ -160,6 +160,16 @@ public static function fatalHandler()
             self::$logger->log(Level::CRITICAL, $exception, $extra);
         }
     }
+    
+    protected static function shouldLogFatal($last_error)
+    {
+        return
+            !is_null($last_error) &&
+            in_array($last_error['type'], self::$fatalErrors, true) &&
+            // don't log uncaught exceptions as they were handled by exceptionHandler()
+            isset($last_error['message']) &&
+            strpos($last_error['message'], 'Uncaught exception') !== 0;
```

Comment:
The `!== 0` should be `!== false` because strpos can return 0 if that is the start of the string, but returns false exactly to indicate not found.